### PR TITLE
[ADD] Manifest wizard

### PR DESCRIPTION
**Pull request by ismaelcj**
I added a wizard where centralise the manifest files. Just abstract, you will have to inherit it in each carrier module.

The manifiest file is a kind of summary of packages which the carrier has to pick up.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Maybe "packing list" is a better term than "manifest"

**Comment by ismaelcj** [comment]
TNT, UPS, DHL and Seur at least call it "manifest".

**Comment by pedrobaeza** [comment]
OK then

**Comment by pedrobaeza** [comment]
Please check access rules:

```
2015-11-02 14:06:05,131 10300 WARNING 3118812-74-a6032c-all openerp.modules.loading: The model manifest.wizard has no access rules, consider adding one. E.g. access_manifest_wizard,access_manifest_wizard,model_manifest_wizard,,1,0,0,0
```

File: base_delivery_carrier_label/wizard/manifest_wizard.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api, exceptions, _
+
+
+class ManifestWizard(models.Model):
```
Comment (yvaucher) [inline base_delivery_carrier_label/wizard/manifest_wizard.py:24]: A wizard must be a `TransientModel`

File: base_delivery_carrier_label/security/ir.model.access.csv [hunk 2]
```diff
@@ -7,3 +7,5 @@ access_shipping_label_user,shipping.label user,model_shipping_label,stock.group_
 access_shipping_label_manager,shipping.label manager,model_shipping_label,stock.group_stock_manager,1,1,1,1
 access_carrier_account_salesman,carrier.account.salesman,model_carrier_account,base.group_sale_salesman,1,0,0,0
 access_carrier_account_sale_manager,carrier.account.sale.manager,model_carrier_account,base.group_sale_manager,1,1,1,1
+access_manifest_wizard_stock_user,access.manifest.wizard.stock.user,model_manifest_wizard,stock.group_stock_user,1,0,0,0
```
Comment (yvaucher) [inline base_delivery_carrier_label/security/ir.model.access.csv:10]: Not access rights to add for a `TransientModel`

File: base_delivery_carrier_label/wizard/manifest_wizard_view.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="manifest_wizard_form" model="ir.ui.view">
+            <field name="name">manifest.wizard.form</field>
+            <field name="model">manifest.wizard</field>
+            <field name="type">form</field>
+            <field name="arch" type="xml">
+                <form string="Manifest" version="7.0">
```
Comment (yvaucher) [inline base_delivery_carrier_label/wizard/manifest_wizard_view.xml:10]: `version="7.0"` to remove

File: base_delivery_carrier_label/wizard/manifest_wizard_view.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            <field name="view_id" ref="manifest_wizard_form"/>
+            <field name="target">new</field>
+        </record>
+
+        <menuitem id="manifest_wizard_menu"
+            action="manifest_wizard_action" string="Manifest"
+            parent="stock.menu_stock_warehouse_mgmt"/>
+
+    </data>
+</openerp>
```
Comment (yvaucher) [inline base_delivery_carrier_label/wizard/manifest_wizard_view.xml:50]: Missing EOL

**Review by yvaucher — CHANGES_REQUESTED** [review]
Can you provide more information about it ?
Explain with which carrier you need it, plus update the README.rst.

**Comment by hparfr** [comment]
Is it different from delivery_carrier_deposit ?

**Comment by pedrobaeza** [comment]
Closing as no news.

**Pull request closed without merging**